src/map/packets_struct.h

```diff
@@ -236,10 +236,11 @@ enum packet_headers {
 #endif
 #if PACKETVER < 20100105
 	vendinglistType = 0x133,
+#elif PACKETVER_MAIN_NUM >= 20200916 || PACKETVER_RE_NUM >= 20200723
```

Comment:
It's probably better to put this at the end instead of middle of the list so the reader can figure out which is the latest more easily.

Reply:
I thought it was not worth it, because I thought you will change it soon, to be defined where the struct is defined. With HEADER_*

Reply:
Yes but we also tend to copy the same ifdefs when doing the conversions, so for sake of clarity keep it sorted.